build.cake

```diff
@@ -544,7 +544,8 @@ void BuildProject(string projectPath, string configuration)
         settings.SetConfiguration(configuration)
         .SetVerbosity(Verbosity.Minimal)
         .WithTarget("Build")
-        .WithProperty("NodeReuse", "false"));
+        .WithProperty("NodeReuse", "false")
+		.WithProperty("Platform", "AnyCPU"));
 }
```

Comment:
On my local system, I had to do this to avoid an error. MsBuild thought I had specified "MCD" as the platform. I found threads online reporting the same thing, going back years! My error was only with the Net Standard build, but since we use AnyCPU for all builds, I figured it couldn't hurt.